packages/frontend/src/common/components/Button.tsx

```diff
@@ -13,7 +13,7 @@ interface LinkProps extends Props {
 }
 
 interface ButtonProps extends Props {
-  onClick: () => void
+  onClick: (param?: any) => void
```

Comment:
What's this?

It's called `param` here, but we're calling the param `inputs` when it's used.

Reply:
I called it `param` here because it's common component so it can be any parameter. In case when I'm using it I send `inputs` because it's true.
I'm happy to improve it if you have better naming idea 🙂

Reply:
After some research I changes it to `onClick: (event: React.MouseEvent<HTMLButtonElement>) => void`.
`onClick: (param: any) => void` wasn't correct :blush: